src/routes/resources/+page.svelte

```diff
@@ -121,14 +125,18 @@
                 <!-- checkboxes -->
                 <div class="flex justify-between gap-2 rounded-full bg-gray-300" >
                     <label class="cursor-pointer py-2 px-3 rounded-full flex items-center gap-2" for={removeWhitespace(tag)}>
-                      <input
-                        type="checkbox"
-                        class="appearance-none cursor-pointer w-6 h-6 bg-white rounded-full checked:bg-black transition duration-200"
-                        bind:checked={filterObject.tags[tag]}
-                        id={removeWhitespace(tag)}
-                        name={removeWhitespace(tag)}
-                      />
-                      <span>{tag}</span>
+                        <input
+                            type="checkbox"
+                            class="appearance-none cursor-pointer w-6 h-6 bg-white rounded-full checked:bg-black transition duration-200"
+                            bind:checked={filterObject.tags[tag]}
+                            id={removeWhitespace(tag)}
+                            name={removeWhitespace(tag)}
+                        />
+                        <span>
+                            {#each Object.entries(tags_count) as [tag_name, tag_count]}
+                                {#if tag == tag_name}{tag_name}: ({tag_count}) {/if}
+                            {/each}
+                        </span>
```

Comment:
I think this would be better? It is a `key: value` structure after all

```suggestion
                        <span>
                            {tag} ({tags_count[tag]})
                        </span>
```

Reply:
yep, look great. gota write that in my notepad :)